src/map/entities/charentity.cpp

```diff
@@ -1655,6 +1645,10 @@ void CCharEntity::OnAbility(CAbilityState& state, action_t& action)
                     }
 
                     addMP((int32)-mpCost);
+                    if (this->GetLocalVar("bpRecastTime") > 0) // This will go away when all smn petskills are handled via jobutils/summoner.lua
```

Comment:
`// This will go away when all smn petskills are handled via jobutils/summoner.lua`
So why isn't this PR adding it there? This C++ block is not necessary anymore. Perhaps this should be removed from this PR so the rest of the PR can be approved and merged sooner

Reply:
Diablos doesn't use the new petskill db table yet

Reply:
ah... interesting that the pet animation hack in c++ wasn't necessary for diabolos